src/map/status.cpp

```diff
@@ -7038,6 +7150,16 @@ static unsigned short status_calc_matk(struct block_list *bl, struct status_chan
 		matk += matk * sc->data[SC_MTF_MATK]->val1 / 100;
 	if(sc->data[SC_2011RWC_SCROLL])
 		matk += 30;
+	if (sc->data[SC_ALMIGHTY])
+		matk += 30;
+	if (sc->data[SC_ULTIMATECOOK])
+		matk += 30;
+	if (sc->data[SC_LIMIT_POWER_BOOSTER])
+		matk += sc->data[SC_LIMIT_POWER_BOOSTER]->val1;
+	if (sc->data[SC_MAGICCANDY])
+		matk += 30;
+	if (sc->data[SC_SKF_MATK])
+		matk += sc->data[SC_SKF_MATK]->val1;
```

Comment:
If I'm not wrong you already defined them in `status_calc_matk` in the line above, I guess it give double bonus. Do you mind to check the bonus?

Reply:
Oh, I saw an example from SC_2011RWC_SCROLL

Thanks!